src/rcpp_constants.cpp

```diff
@@ -1586,6 +1586,13 @@ String LUC_EMISSIONS() {
 return D_LUC_EMISSIONS;
 }
 
+//' @rdname carboncycle
+//' @export
```

Comment:
Add @describeIn section?

Reply:
I'm not an expert but the [documentation](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html) says "use **either** @rdname or @describeIn to control where method documentation goes". And everything here as `@rdname` tags so I think we're good?